design-tokens/tokens/layer1/base.json

```diff
@@ -55,7 +55,7 @@
           "type": "dimension"
         }
       },
-      "size": {
+      "width": {
```

Comment:
ouf this is a big naming change, it's used everywhere... do we really need to make a difference between width and height on layer 1 ?

Reply:
Discussed live with @gregaubert